### PR TITLE
Add ONNX docs to onnx.ai

### PR DESCRIPTION
**Pull request by natke**
Autogenerate and autopublish ONNX documentation to the https://onnx.ai website

Docs are staged here: https://natke.github.io/onnx (and once this PR is merged will autopublish to https://onnx.ai/onnx)

### CODE REVIEW

File: docs/docsgen/source/onnx_sphinx.py [hunk 1]
```diff
@@ -0,0 +1,824 @@
...
+            renderNothingWhenEmpty: false,
+        };
+        var diff2htmlUi = new Diff2HtmlUI(targetElement, diffString, configuration);
+        diff2htmlUi.draw();
+        diff2htmlUi.highlightCode();
+        });
+        </script>
+        """
+        )
+    )
```
[the remark this answers is not in the document]
Comment (natke) [reply docs/docsgen/source/onnx_sphinx.py:85]: @xadupre Can you please have a look at this warning?

**Comment by AlexandreEichenberger** [comment]
Suggestion: the text in the diff is in a low contrast grey that is hard to read. Please keep it in easy to read black:

image

**Comment by AlexandreEichenberger** [comment]
Minor nit: when we click on a link that goes to the end of a page (e.g. Table on the onnx operator), the title is not at the top of the page , probably because it is the end of the page.
Adding blank carriage return may be a simple fix

**Comment by prasanthpul** [comment]
* some of the ops have "or" as a subheading. seems like a formatting error?

* "expect" as a heading on the main operator page is odd. it's the function used, but doesn't make sense as a header.

* can the op tables be used as the operator TOC? its basically the same info, except for the direct links to older versions

**Comment by jackwish** [comment]
This is awesome! A quick question, why title "Python Documentation of ONNX" at landing page (similar issue of `onnx_python` for the path `docs/docsgen/source/onnx_python/`) while it doesn't seem to be python specific?

**Comment by prasanthpul** [comment]
superceded by #4561

**Pull request closed without merging**